lighthouse-core/test/audits/byte-efficiency/uses-long-cache-ttl-test.js

```diff
@@ -116,6 +116,11 @@ describe('Cache headers audit', () => {
   });
 
   it('respects expires/cache-control priority', () => {
+    // Stub Date.now so the test is not sensitive to timing.
+    const now = Date.now();
```

Comment:
could make the argument that this should be in a beforeAll. thoughts?

Reply:
I would make that argument yes :)